apps/nextjs-example/tailwind.config.ts

```diff
@@ -0,0 +1,20 @@
+import type { Config } from "tailwindcss";
```

Comment:
Wondering if we gonna ever use tailwind in our production code ;)

Reply:
yeah. I figured this was a fine place to start 😄